Master

akeranen opened the pull request:
First stab at using array root SenML element

akeranen committed:
Merge pull request #1 from fluffy/master

Changes from fluffy repo

akeranen committed:
Synced with XML from -01 with small fixes

Editorial fixes from Carsten
Fixed temperature XML/EXI example

akeranen committed:
Merge remote-tracking branch 'fluffy/master'

Conflicts:
	draft-jennings-core-senml-02.md

akeranen committed:
First stab at root SenML array

cabo commented:
How about the idea that a base object in position 2, 4, 6 is a merge-patch, updating the state resulting from the previous state base object?

Why is there no base value (yes, bv has to be renamed)?

akeranen commented:
The merge-patch text is coming later today. I just wanted to pull-req the first set of updates to avoid conflicts. But good point with base value. Probably should have that too for completeness.

akeranen committed:
JSON merge patch and small fixes

Added text about JSON merge patch format for updating base values.
Clarified that Units is Optional. Small editorial fixes.

akeranen commented:
Now including the JSON merge patch text.
